test/sql/pragma/test_custom_profiling_settings.test

```diff
@@ -0,0 +1,32 @@
+# name: test/sql/pragma/test_custom_profiling_settings.test
+# description: Test PRAGMA custom_profiling_settings
+# group: [pragma]
+
+# enable profiling cannot be called
+statement ok
+PRAGMA custom_profiling_settings='__TEST_DIR__/test.json'
```

Comment:
How do you mean that it cannot be called? Why does this statement succeed?

Reply:
I think I moved the test but forgot to move the comment....